guides/common/modules/proc_assigning-a-puppet-class-to-an-individual-host.adoc

```diff
@@ -3,10 +3,10 @@
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Click on the *Edit* button of the host you want to add the `ntp` Puppet class to.
+. Click *Edit* of the host you want to add the `ntp` Puppet class to.
```

Comment:
```suggestion
. Click *Edit* for the host you want to add the `ntp` Puppet class to.
```

Reply:
```suggestion
. Locate the host you want to add the `ntp` Puppet class to  and click *Edit*.
```

Reply:
Neither seems great. Can you think of something better?

Reply:
I like the second suggestion and applied it.